### PR TITLE
fix stack usage in vcmp method (cmp overload)

### PR DESCRIPTION
**Pull request by haarg**

### CODE REVIEW

File: vutil/vxs.inc [hunk 1]
```diff
@@ -312,7 +312,7 @@ VXS(version_normal)
 VXS(version_vcmp)
 {
      dXSARGS;
-     if (items < 1)
+     if (items < 2)
 	 croak_xs_usage(cv, "lobj, ...");
```
Comment (Leont) [inline vutil/vxs.inc]: Ideally the croak message also gets updated to take a `robj`

File: vutil/vxs.inc [hunk 2]
```diff
@@ -322,7 +322,7 @@ VXS(version_vcmp)
 	       SV	*rs;
 	       SV	*rvs;
 	       SV * robj = ST(1);
-	       const IV	 swap = (IV)SvIV(ST(2));
+	       const int swap = items > 2 ? SvTRUE(ST(2)) : 0;
```
Comment (atoomic) [inline vutil/vxs.inc:325]: @haarg should not we customize the version in blead with this fix?
ping @ZenoParadoxus

Comment (Leont) [reply vutil/vxs.inc:325]: I just took care of the final permissions, I'll put a new release out

Comment (atoomic) [reply vutil/vxs.inc:325]: side topic: I wonder if this should be a `dist` and not a `cpan` module on blead

**Pull request merged**